homie/examples/light.rs

```diff
@@ -0,0 +1,39 @@
+use homie::{Datatype, HomieDevice, Node, Property};
+use rumqttc::MqttOptions;
+use std::error::Error;
+use tokio::try_join;
+
+#[tokio::main(core_threads = 2)]
+async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
+    pretty_env_logger::init();
+
+    let mqttoptions = MqttOptions::new("homie_example", "test.mosquitto.org", 1883);
+
+    let mut builder = HomieDevice::builder("homie/example", "Homie light example", mqttoptions);
+    builder.set_update_callback(|node_id, property_id, value| async move {
+        println!("{}/{} is now {}", node_id, property_id, value);
+        true
```

Comment:
Naked booleans make me feel uncomfortable. They're almost always the wrong answer.

Maybe the callback could return Ok(value) for success or Err for failure?

Reply:
What would the error type be? Or should it just be an `Option`?